lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -214,17 +215,17 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
         const userAuthority = requestAuthority ? requestAuthority : this.config.auth.authority;
 
         // fall back to the authority from config
-        const builtAuthority = Authority.generateAuthority( userAuthority, requestAzureCloudOptions || this.config.auth.azureCloudOptions);
+        const builtAuthority = Authority.generateAuthority(userAuthority, requestAzureCloudOptions || this.config.auth.azureCloudOptions);
         this.logger.verbose("Creating discovered authority with configured authority", this.correlationId);
-        return await AuthorityFactory.createDiscoveredInstance(builtAuthority, this.config.system.networkClient, this.browserStorage, authorityOptions)
+        return await AuthorityFactory.createDiscoveredInstance(builtAuthority, this.config.system.networkClient, this.browserStorage, authorityOptions, undefined, requestCorrelationId, this.performanceClient)
             .then((result: Authority) => {
                 getAuthorityMeasurement.endMeasurement({
```

Comment:
Instead of adding this here, can we rather collect all network related telemetry in the INetworkModule implementation - in this case it will be `XhrClient.ts` and `FetchClient.ts`? Probably will make it centralized and independent of which network call is made.

Reply:
Do you mean to get rid of StandardInteractionClientGetDiscoveredAuthority submeasurement as a whole from here ? @sameerag

Reply:
Keeping it here as discussed.